recipes/newrelic/apm/php/debian.yml

```diff
@@ -0,0 +1,502 @@
+              # FPM will be the one to use and `newrelic-install.sh` only
+              # installs the `newrelic.ini` in one web location.
+              #
+              echo "$nonpriv_user" >> {{.TMP_INSTALL_DIR}}/nonpriv_users.txt
+            fi
+          done
+
+    add_gnupg2_curl_if_required:
+      cmds:
+        - |
+          if [ $(({{.DEBIAN_VERSION}})) -ge 10 ]; then
+            # If possible it would be great to note if these were already
+            # installed, and if not, to remove them in a later cleanup step
+            sudo apt-get install gnupg2 curl -y
+          fi
+      vars:
+        DEBIAN_VERSION:
+          sh: awk -F= '/VERSION_ID/ {print $2}' /etc/os-release
+
+    install_tarball:
+      cmds:
+        - |
+          echo -e "{{.ARROW}}Installing New Relic PHP Agent{{.GRAY}}"
+          echo -e "{{.WHITE}}Downloading newest PHP Agent Release{{.GRAY}}"
+          cd {{.TMP_INSTALL_DIR}}
+          # Remove old log tarballs to ensure we don't accidentally use it later
+          rm -f /tmp/nrinstall* 2>/dev/null
+
+          # Versions of the form MAJ.MIN.PATCH.BUILD
+          VERSION_REGEX='[1-9][0-9]\?\(\.[0-9]\+\)\{3\}'
+          # PHP Agent current release tarball listing. '/' is crucial
+          # for treating directory as an HTML directory listing
+          RELEASE_URL='https://download.newrelic.com/php_agent/release/'
+          AGENT_VERSION="$(curl -s "$RELEASE_URL" | grep --only-match "$VERSION_REGEX" | head -n1)"
+          echo -e "{{.GRAY}}AGENT VERSION: $AGENT_VERSION{{.NOCOLOR}}"
+          if [ -z "${AGENT_VERSION}" ]; then
+            echo -e "{{.RED}}Unable to determine current PHP Agent version from New Relic's downloads site.{{.NOCOLOR}}"
+            # Exit with Agent install failure code exit(16)
+            exit 16
+          else
+            echo -e "{{.WHITE}}Found agent version: {{.CYAN}}$AGENT_VERSION{{.GRAY}}"
+          fi
+
+          AGENT="newrelic-php5-$AGENT_VERSION-linux"
+          AGENT_TARBALL="$AGENT.tar.gz"
+          # Lack of '/' is important. See above comment where $RELEASE_URL is
+          # defined. An extraneous '/' here treats the URL as an HTML doc.
+          curl -s "$RELEASE_URL$AGENT_TARBALL" -o "$AGENT_TARBALL"
+          gzip -dc "$AGENT_TARBALL" | tar xf -
+          pushd "$AGENT" > /dev/null
+          echo -e "{{.WHITE}}Running PHP Agent installer{{.GRAY}}"
+          NR_INSTALL_USE_CP_NOT_LN=1 NR_INSTALL_SILENT=true NR_INSTALL_KEY="{{.NEW_RELIC_LICENSE_KEY}}" ./newrelic-install install
+          popd > /dev/null
+
+    install_deb:
+      cmds:
+        - |
+          echo -e "{{.ARROW}}Installing New Relic PHP Agent package{{.GRAY}}"
+          echo 'deb http://apt.newrelic.com/debian/ newrelic non-free' | sudo tee /etc/apt/sources.list.d/newrelic.list
+          curl -s https://download.newrelic.com/548C16BF.gpg | sudo apt-key add -
+          sudo apt-get update
```

Comment:
@zsistla it would be good to add the option to silence any invalid check as it's done in the infra recipe https://github.com/newrelic/open-install-library/blob/main/recipes/newrelic/infrastructure/debian.yml#L216

Reply:
Will do, thanks for the link!